sphinxext/opengraph/__init__.py

```diff
@@ -134,12 +135,12 @@ def get_tags(
             image_url_parsed = urlparse(image_url)
             if not image_url_parsed.scheme:
                 # Relative image path detected, relative to the source. Make absolute.
-                if config["ogp_image"]:
+                if first_image:
+                    root = page_url
+                else:
```

Comment:
```suggestion
                else: # ogp_image is set
```

Reply:
The fact that the workaround is only for one specific case should remain documented.